generate_chip.py

```diff
@@ -0,0 +1,345 @@
+"""
+Generate the following packages:
+
+- Chip resistors SMT
+
+"""
+from os import path, makedirs
+from typing import Iterable, Optional, Dict, Any  # noqa
+from uuid import uuid4
+
+from common import now, init_cache, save_cache
+from common import format_float as ff, format_ipc_dimension as fd
+from common import generate_courtyard, indent
+
+
+generator = 'librepcb-parts-generator (generate_chip.py)'
+
+line_width = 0.25
+line_width_thin = 0.15
+line_width_thinner = 0.1
```

Comment:
This is too thick for the smallest packages:

```suggestion
line_width_thinner = 0.05
```

Reply:
This is also used for silkscreen, and IIRC most manufacturers will probably have issues with silkscreen below 0.1mm.

For example, according to http://blog.optimumdesign.com/toward-a-more-rational-silkscreen, depending on the technique, minimum width is between 0.15 and 0.075 mm.

But we can have different sizes for placement and doc layers.

Reply:
Ah, no, you're right! Sorry, I'm on my mobile :)

Reply:
Yep, 0.05 looks good!